Shared/Samples/Show portal user info/ShowPortalUserInfoView.swift

```diff
@@ -0,0 +1,297 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct ShowPortalUserInfoView: View {
+    /// The data model that helps determine the view.
+    @State private var model = Model()
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        VStack {
+            // Shows field for custom portal urls as well as Sign In / Out.
+            PortalDetailsView(
+                url: $model.portalURLString,
+                model: $model,
+                // Update the model when the portal URL is changed by the user.
+                onSetUrl: {
+                    model.portalURLString = $0
+                },
+                // Sign out the user.
+                onSignOut: {
+                    Task {
+                        await model.signOut()
+                    }
+                },
+                // Load the user's portal data when sign in is triggered.
+                onLoadPortal: {
+                    Task {
+                        await loadUser()
+                    }
+                }
+            )
+            // If there is no current user, show that the user profile will display here when complete.
+            if model.portalUser == nil {
+                ContentUnavailableView(
+                    "Portal User Information",
+                    systemImage: "person.crop.circle.dashed",
+                    description: Text("Your portal user information will be displayed here.")
+                )
+                // Otherwise show the user information that was loaded.
+            } else {
+                InfoScreen(model: $model)
+            }
+            Spacer()
+        }
+        .frame(maxHeight: .infinity)
```

Comment:
I'm not sure if this is having any effect(?). Instead, it might be better to set a max width here, because the text field and list dividers are back to full window width on Catalyst.

Reply:
Are you looking for something like this? 
<img width="1028" height="884" alt="Screenshot 2025-07-23 at 10 58 15 AM" src="https://github.com/user-attachments/assets/a971a18b-4fda-42bb-9923-096f16a82b99" />